core/src/test/java/org/elasticsearch/bootstrap/JarHellTests.java

```diff
@@ -114,7 +112,7 @@ public void testDirAndJar() throws Exception {
     public void testWithinSingleJar() throws Exception {
         // the java api for zip file does not allow creating duplicate entries (good!) so
         // this bogus jar had to be constructed with ant
```

Comment:
This comment is out of date; we produce these with https://github.com/jasontedor/duplicate-classes/ now (the previous mechanism for building the bogus jars produced invalid jars which blew up animal-sniffer).